TileStache/Goodies/VecTiles/transform.py

```diff
@@ -1563,6 +1670,76 @@ def generate_label_features(
         return label_feature_layer
 
 
+def generate_address_points(
+        feature_layers, zoom, source_layer=None, start_zoom=0):
+    """
+    Generates address points from building polygons where there is an
+    addr:housenumber tag on the building. Removes those tags from the
```

Comment:
Yes, there are lots of changes from these PRs... is the appropriate thing to include updates to the changelog in the PR, or just flag them for the changelog to be updated when pushing & tagging a new version?